### ISSUE TITLE
add jenkins to webhook

### ISSUE THREAD

**Issue by ikozhukhov** [gogs/gogs#5833]
add jenkins CI to list of webhook

**Comment by unknwon** [comment]
Hi, what make this Jenkins plugin not enough? https://wiki.jenkins.io/display/JENKINS/Gogs+Webhook+Plugin

**Comment by ikozhukhov** [comment]
have to add jenkins link in list of webhooks

**Comment by unknwon** [comment]
Thanks for the follow up!

>have to add jenkins link in list of webhooks where we can see slack and others

Sorry I can't understand this sentence.

**Comment by ikozhukhov** [comment]
well, where you go to configuration of webhook in gogs UI you can see just slack and others, you have no jenkins in list.
we need ability configuration of webhook under gogs and under jennkins for different usecases

**Comment by unknwon** [comment]
I see, can you directly answer to my question: what make this Jenkins plugin not enough? wiki.jenkins.io/display/JENKINS/Gogs+Webhook+Plugin

**Comment by ikozhukhov** [comment]
well, let me try take a look type 'GOGS' for it. i tried to look application 'jenkins' in list with template
https://cl.ly/c67e51f1c5e8
take a look picture - i tried to find 'jenkins' in this list with template.

**Comment by unknwon** [comment]
Let me rephrase my question: why can you just use https://wiki.jenkins.io/display/JENKINS/Gogs+Webhook+Plugin?

**Comment by ikozhukhov** [comment]
i didn't try this plugin yet - i'll try. i just try to learn GOGS : how it use with CI build based on jenkins jobs

**Comment by unknwon** [comment]
Hmm OK, Gogs currently does not have CI build status support.

**Comment by ikozhukhov** [comment]
yes :)

**Comment by unknwon** [comment]
GitLab is a good one, if you have enough server resources.

**Comment by ikozhukhov** [comment]
but gitlab is not single binary like gogs and gitea ;)

**Comment by unknwon** [comment]
Docker to rescue!

**Comment by ikozhukhov** [comment]
i'm using dilos with zfs and zones. host a single app is enough for my needs
